content/en/docs/setup/install/operator/index.md

```diff
@@ -75,11 +73,197 @@ accessible to `istioctl` by using this command:
+1. [Replica count](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+1. [HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+1. [PodDisruptionBudget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#how-disruption-budgets-work)
+1. [Pod annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+1. [Service annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+1. [ImagePullPolicy](https://kubernetes.io/docs/concepts/containers/images/)
+1. [Priority class name](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass)
+1. [Node selector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector)
+1. [Affinity and anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
+
+All of these K8s settings use the K8s API definitions, so [K8s documentation](https://kubernetes.io/docs/concepts/) can be used for reference. 
+
+### Configure the feature or component settings
+After you indentify the name of the feature or component from the previous list, you can use the API to set the values using a configuration overlay file.
+
+For example, use this overlay file ([samples/pilot-k8s.yaml](https://github.com/istio/operator/blob/release-1.3/samples/pilot-k8s.yaml))
+to adjust the resources and HPA scaling settings for Pilot, which is a part of the TrafficManagement feature:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  trafficManagement:
+    components:
+      pilot:
+        k8s:
+          resources:
+            requests:
+              cpu: 1000m # override from default 500m
+              memory: 4096Mi # ... default 2048Mi
+          hpaSpec:
+            maxReplicas: 10 # ... default 5
+            minReplicas: 2  # ... default 1
+{{< /text >}}
+
+## Customize Istio settings using the Helm API
+
+The [Helm API](https://istio.io/docs/reference/config/installation-options/) is available as part of the Operator API.
+You can access it as a top level values field in IstioControlPlane
+(for [global settings](https://istio.io/docs/reference/config/installation-options/#global-options))
+and per-component values fields for each Istio component. 
+
+For example, here's a YAML file that configures some global and Pilot settings through the Helm API:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  trafficManagement:
+    components:
+      pilot:
+        values:
+          traceSampling: 0.1 # override from 1.0
+
+  # global Helm settings
+  values:
+    monitoringPort: 15050
+{{< /text >}}
+
+Note that some parameters will temporarily exist in both the Helm and IstioControlPlane APIs - for example, K8s resources,
```

Comment:
do not use the word Note.  :) . I'd add a warning callout here and place it at the beginnning of the Helm API section.

Reply:
I don't think this is worthy of a callout. I've removed the word Note.